pyanaconda/ui/webui/__init__.py

```diff
@@ -141,10 +141,16 @@ def _run_webui(self):
         # FIXME: We probably want to move this to early execution similar to what we have on live
         profile_name = FIREFOX_THEME_DEFAULT
 
-        proc = startProgram(["/usr/libexec/webui-desktop",
-                             "-t", profile_name, "-r", str(int(self.remote)),
-                             "/cockpit/@localhost/anaconda-webui/index.html"],
-                            reset_lang=False)
+        log_file_path = "/tmp/anaconda.log"
+
+        with open(log_file_path, "a") as log_file:
+            proc = startProgram(
```

Comment:
Please use `execWithRedirect` instead without a log file specification. That will log the output automatically for you and you don't have to deal with logs.

Reply:
I may have found the reason. From execWithRedirect it is not possible to get for example the PID of the process, which is then important for killing the process